engine/pom.xml

```diff
@@ -322,6 +322,51 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>add-module-info</goal>
+                        </goals>
+                        <configuration>
+                            <module>
+                                <moduleInfo>
+                                    <name>${java.module.name}</name>
+                                    <requires>
+                                        <!-- jboss annotations are getting pulled even though they are for compilation only -->
+                                        !jboss.logging.annotations;
+                                        *;
+                                    </requires>
+                                    <exports>
+                                        org.hibernate.validator.*.logging to org.jboss.logging, org.hibernate.validator.cdi;
+                                        org.hibernate.validator.internal to org.hibernate.validator.cdi;
+                                        org.hibernate.validator.internal.* to org.hibernate.validator.cdi;
+                                        org.hibernate.validator.*.internal.* to org.hibernate.validator.cdi;
+                                        *;
```

Comment:
Might as well export `org.hibernate.validator.*`, no? It's not like we don't trust our own modules...

Reply:
I think if we export `org.hibernate.validator.*` to `org.hibernate.validator.cdi` then we'll be exporting only to that module and all the public packages won't be exported to any other module (even public packages)